src/tapis-ui/_common/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { range } from 'tapis-ui/utils/range';
+import { NavLink } from 'react-router-dom';
+import styles from './Breadcrumbs.module.scss';
+
+export type BreadcrumbType = {
+  to: string;
+  text: string;
+};
+
+type BreadcrumbsProps = {
+  breadcrumbs: Array<BreadcrumbType>;
+};
+
+const Breadcrumbs: React.FC<BreadcrumbsProps> = ({ breadcrumbs }) => {
+  let truncatedIndices: Array<number> = [];
+
```

Comment:
It is probably more efficient in this case to render the first two elements, the ellipsis, and then the last two elements. that way you don't have to generate an extra array and use the includes function.

Reply:
Reduced complexity: Got first 2 Breadcrumbs, appended the ellipsis breadcrumb, the the last 2 breadcrumbs in the breadcrumbs array.